cmd/limactl/editflags/editflags.go

```diff
@@ -66,6 +66,11 @@ func registerEdit(cmd *cobra.Command, commentPrefix string) {
 
 	// negative performance impact: https://gitlab.com/qemu-project/qemu/-/issues/334
 	flags.Bool("video", false, commentPrefix+"Enable video output (has negative performance impact for QEMU)")
+
+	flags.Float32("disk", 0, commentPrefix+"Disk size in GiB") // colima-compatible
+	_ = cmd.RegisterFlagCompletionFunc("memory", func(*cobra.Command, []string, string) ([]string, cobra.ShellCompDirective) {
```

Comment:
I know you just moved the call, but it looks like it is incorrect:

```suggestion
	_ = cmd.RegisterFlagCompletionFunc("disk", func(*cobra.Command, []string, string) ([]string, cobra.ShellCompDirective) {
```

Reply:
Thanks so much! I've already squashed the commit.